app/components/projects/published-item/component.stories.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Story } from '@storybook/react/types-6-0';
+import Item, { PublishedItemProps } from './component';
+
+export default {
+  title: 'Components/Projects/Item',
+  component: Item,
+  argTypes: {},
+};
+
+const Template: Story<PublishedItemProps> = ({ ...args }: PublishedItemProps) => (
+  <Item {...args} />
+);
+
+export const Default = Template.bind({});
+Default.args = {
+  id: 1,
```

Comment:
Can you check the default args you are setting for this item? I think you need to update them

Reply:
Updated and rendered on page.